dataland-frontend/src/components/resources/taxonomy/search/EuTaxoSearchBar.vue

```diff
@@ -1,210 +1,127 @@
 <template>
   <MarginWrapper>
     <div class="grid">
-      <div class="col-8 text-left" v-if="!scrolled">
+      <div class="col-8 text-left">
         <span class="p-fluid">
           <span class="p-input-icon-left p-input-icon-right">
             <i class="pi pi-search" aria-hidden="true" style="z-index: 20; color: #958d7c" />
             <i v-if="loading" class="pi pi-spinner pi-spin" aria-hidden="true" style="z-index: 20; color: #e67f3f" />
             <i v-else aria-hidden="true" />
             <AutoComplete
-              v-model="selectedCompany"
-              :suggestions="filteredCompaniesBasic"
-              @focus="focused"
-              @focusout="unfocused"
-              @complete="searchCompany($event)"
-              placeholder="Search company by name or PermID"
-              inputClass="h-3rem"
+              :suggestions="autocompleteArrayDisplayed"
+              :name="taxoSearchBarName"
+              :modelValue="modelValue"
               ref="autocomplete"
+              inputClass="h-3rem"
               field="companyName"
               style="z-index: 10"
-              name="eu_taxonomy_search_input"
-              @keyup.enter="handleQuery"
+              placeholder="Search company by name or PermID"
+              @input="handleInput"
+              @complete="searchCompanyName"
+              @keyup.enter="handleCompanyQuery"
               @item-select="handleItemSelect"
```

Comment:
Die Stelle ist aktuell kaputt.
Wenn man mit den Pfeiltasten in den Autocomplete Vorschlägen etwas auswählt und enter drückt, dann konkurrieren die events "item-select" und "keyup.enter" miteinander, weil beide einen router.push ausführen!

Reply:
Reparieren + Test dafür bauen?

Reply:
Wurde repariert.   Jedoch noch keinen Test geschrieben.

Reply:
Ist immer noch kaputt.
Wenn das Dropdown offen ist, reagiert er bei Drücken von Enter nicht mit einem "submit"!

Reply:
Endergebnis nach Gespräch:
Wir "akzeptieren" den Bug erstmal und stellen alles wieder auf keyup.enter um anstatt submit.
Issue bei primevue erstellen, dass man auf submit reagieren kann. Sobald gefixt => Wir fixen den Bug

Pfeiltastenproblem-Test ist im Jira Ticket gepastet und bereits ready zum integrieren, sobald wir es wollen.

Reply:
ok